docs/source/development.rst

```diff
@@ -84,7 +84,7 @@ Let's now try to run some tests.
 That looks good. Notice a couple of things:
 
 If you're a developer, you probably want to start with only one MPI rank. Beware of the size of the simulations, though. You probably want to solve a problem with a minimum reproducer having a couple of cells only.
-**"Special available. Replacing binary."** informs us that it detected a "special" binary and is going to use it (instead of attempting to load libnrnmech.so). If you need to disable that, set the env variable: export neurodamus_special=1
+**"Special available. Replacing binary."** informs us that it detected a "special" binary and is going to use it (instead of attempting to load libnrnmech.so). If you need to disable that, set the env variable: export NEURODAMUS_SPECIAL=1
```

Comment:
was this needed? It may cause regressions

Reply:
I had the same question; since it seems to be a `development` thing, I hope it's safe; I agree with the lint that env vars should be capitalized, which is why I'd like to apply it, but if you think it's safer not to, I'm ok reverting that.